### PR TITLE
feat(docs): add qabot to jina now docs

### PR DESCRIPTION
**Pull request by winstonww**

### CODE REVIEW

File: docs/_templates/page.html [hunk 1]
```diff
@@ -227,6 +227,28 @@
...
+        target="_self"
+        orientation="bottom-right"
+        title="Jina Now"
+        description="Bring your search case into the cloud in minutes."
+        show-tip>
+         <template>
+          <dl>
+           <dt>You can ask questions about our docs. Try:</dt>
+           <dd>What is jina now?</dd>
+           <dd>How to install now?</dd>
```
Comment (makram93) [inline docs/_templates/page.html:242]:
```suggestion
           <dd>What is Jina NOW?</dd>
           <dd>How to install NOW?</dd>
```

File: docs/_templates/page.html [hunk 2]
```diff
@@ -227,6 +227,28 @@
...
+           <dd>Where apps are supported?</dd>
```
Comment (makram93) [inline docs/_templates/page.html:243]:
```suggestion
           <dd>What apps are supported?</dd>
```

**Review by makram93 — COMMENTED** [review]
Some minor comments otherwise looks good to me :)

File: docs/_templates/page.html (hunk 1 above)
Comment (winstonww) [reply docs/_templates/page.html]: Addressed.

File: docs/_templates/page.html (hunk 2 above)
Comment (winstonww) [reply docs/_templates/page.html]: Addressed, thanks.

**Pull request merged**